### PR TITLE
schema's creation

### PR DESCRIPTION
**Pull request by ksvirkou-hubspot**

### CODE REVIEW

File: php/src/Commands/Schemas/CreateCommand.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        $schema->setRequiredProperties($properties['required']);
+
+        $response = $hubspot->crm()->schemas()->CoreApi()->create($schema);
+
+        $io->info($response);
+
+        return SchemasCommand::SUCCESS;
+    }
+
+    protected function askProperties(SymfonyStyle $io): array
```
Comment (atanasiuk-hubspot) [inline php/src/Commands/Schemas/CreateCommand.php:60]: Would `askForProperties` and `askForOptions` naming sound more natural?

File: php/src/Commands/Schemas/SchemasCommand.php [hunk 2]
```diff
@@ -17,4 +17,30 @@ protected function addObjectTypeIdToCommand(): void
             )
         ;
     }
+
+    protected function getNamesValidator(): callable
+    {
+        return function (string $string): string {
+            if (empty($string)) {
```
Comment (atanasiuk-hubspot) [inline php/src/Commands/Schemas/SchemasCommand.php:24]: Can we make `getNotEmptyValidator` reusable here, so no code duplication?

Comment (atanasiuk-hubspot) [reply php/src/Commands/Schemas/SchemasCommand.php:24]: and treat `namesValidator` as a combination of `notEmptyValidator`, `notBlankValidator`, etc

File: php/src/Commands/Schemas/CreateCommand.php (hunk 1 above)
Comment (ksvirkou-hubspot) [reply php/src/Commands/Schemas/CreateCommand.php]: done

File: php/src/Commands/Schemas/SchemasCommand.php (hunk 2 above)
Comment (ksvirkou-hubspot) [reply php/src/Commands/Schemas/SchemasCommand.php:24]: I can't do it

**Review by atanasiuk-hubspot — APPROVED** [review]

**Pull request merged**